packages/editor/src/components/start-page-options/index.js

```diff
@@ -129,13 +26,19 @@ export default function StartPageOptions() {
 			! preferencesModalActive &&
 			! isEditedPostDirty() &&
 			isEditedPostEmpty() &&
-			TEMPLATE_POST_TYPE !== getCurrentPostType()
+			'page' === getCurrentPostType()
 		);
 	}, [] );
+	const { setIsInserterOpened } = useDispatch( editorStore );
+
+	useEffect( () => {
+		if ( shouldEnable ) {
+			setIsInserterOpened( {
+				tab: 'patterns',
+				category: 'core/starter-content',
+			} );
+		}
+	}, [ shouldEnable, setIsInserterOpened ] );
```

Comment:
I think the way this is built, it won't reopen the panel if you use the command palette to "create a new page" when you're already creating a new page. I think we should probably add the postId as a dependency.

Reply:
In my testing this is not the case - when you create a new page using the command palette, zoom out is invoked again.

Reply:
Actually on further testing, this is an issue.

Reply:
looks good to me!

Reply:
I dont think you should remove this change though, I think we need both.